### PR TITLE
Better entitled

### PR DESCRIPTION
**Pull request by federicomarini**
... to have the title of the app handled correctly.

While at it, a couple of other fixes

### CODE REVIEW

**Comment by federicomarini** [comment]
GHA currently failing because of the missing `qpdf` on MacOS otherwise looks good for you all to review it @LTLA @kevinrue @csoneson

File: R/iSEE-main.R [hunk 1]
```diff
@@ -556,6 +554,29 @@ iSEE <- function(se,
...
+#' Credits to `@RiboRings` for identifying the issue with the previous approach 
+#' based on `ifelse`. Thanks Giulio!
+#'
+#' @param title Character, which can be any value accepted by `dashboardHeader`
+#'
+#' @return The processed title element
+#' 
+#' @rdname INTERNAL_set_title
+.set_title <- function(title) {
+  if( is.null(title) ) {
```
Comment (csoneson) [inline R/iSEE-main.R]: The spacing here is inconsistent with the rest of the package - I'd rather do 
`if (is.null(title)) {`

**Review by csoneson — APPROVED** [review]
Apart from a spacing inconsistency, LGTM!

**Comment by csoneson** [comment]
Re: the `qpdf` issue: https://forum.posit.co/t/r-cmd-check-fails-on-macos-latest-release-r-lib-actions-setup-r-v2/195084

**Comment by federicomarini** [comment]
> Re: the `qpdf` issue: https://forum.posit.co/t/r-cmd-check-fails-on-macos-latest-release-r-lib-actions-setup-r-v2/195084

Ok, then we simply wait a bit 😉

File: R/iSEE-main.R (hunk 1 above)
Comment (federicomarini) [reply R/iSEE-main.R]: oh good catch, odd I did not catch that already. thanks!

Comment (federicomarini) [reply R/iSEE-main.R]: fixed that now

**Comment by federicomarini** [comment]
Just checking again, and I see that the GHA all passed.
Merging in 😉

**Comment by kevinrue** [comment]
pushed upstream

**Pull request merged**